### ISSUE TITLE
dblclick event not firing.

### ISSUE THREAD

**Issue by prakharpbuf** [katspaugh/wavesurfer.js#2655]
Hi! The dblclick event on the wavesurfer instance is not firing for some reason.

```
wavesurfer.on('dblclick', () => {
    console.log("double clicked");
});
```

**Comment by prakharpbuf** [comment]
listening for dblclick on the drawer worked:

```
wavesurfer.drawer.on('dblclick', () => {
    console.log("double clicked");
});
```

**Comment by katspaugh** [comment]
There’s no equivalent of drawer in v7. Let me reopen this issue to expose a dblclick event in the main class.

**Comment by katspaugh** [comment]
@psavva I've added a dblclick event in #3196 and released it in v7.3.2.
